justfile

```diff
@@ -37,6 +37,16 @@ stream-test args:
 node-test test_name:
   just stream-test //src/workerd/api/node:tests/{{test_name}}-nodejs-test
 
+# e.g. just wpt-test urlpattern
+wpt-test test_name:
+  just stream-test //src/workerd/api/wpt:{{test_name}}
+
+lldb-wpt-test test_name: build
+  cd bazel-bin/src/workerd/api/wpt/{{test_name}}.runfiles/workerd/src/workerd/api/wpt; lldb ../../../workerd/server/workerd  -- test {{test_name}}.wd-test --experimental --directory-path=TEST_TMPDIR=/tmp
```

Comment:
[nit] `working-directory` (https://just.systems/man/en/working-directory.html) might be a better option than `cd`

Reply:
nice catch! i love it. thank you for the review.

Reply:
Unfortunately, it doesn't seem like you can use arguments like `{{test_name}}` with the working-directory directive.